### PR TITLE
Adds moodlet for moth wings burning off

### PR DESCRIPTION
**Pull request by ArcaneDefence**
## About The Pull Request
Gives moths a bad moodlet when their wings burn off that is the same change as dismemberment, but 2 minutes longer, so it's not just the mood changes of being on fire and having been hot.

Mothblocks said someone could do this, and I guess no one took that offer up before I got home.

## Why It's Good For The Game
Better mechanical representation of the suffering inflicted upon mothpeople improves the experience for everybody.

## Changelog
:cl:
add: Mothpeople now actually have their mood worsened when their wings burn off.
/:cl:

### CODE REVIEW

File: code/datums/mood_events/generic_negative_events.dm [hunk 1]
```diff
@@ -310,3 +310,8 @@
 	description = "<span class='warning'>All the fish are dead...</span>\n"
 	mood_change = -3
 	timeout = 90 SECONDS
+
+/datum/mood_event/burnt_wings
+	description = "<span class='boldwarning'>My wings were RUINED!!</span>\n"
```
Comment (Jared-Fogle) [inline code/datums/mood_events/generic_negative_events.dm:315]: I don't like the wording on this, "were" is definitely not correct. Here's my recommendation based on the in game text, feel free to play with it:

```suggestion
	description = "<span class='boldwarning'>MY PRECIOUS WINGS!!</span>\n"
```

Comment (ArcaneDefence) [reply code/datums/mood_events/generic_negative_events.dm:315]: Were was thinking about how there's nothing stopping it from happening and the situation changing, same as if someone gets an arm severed and then replaced with a prosthetic. 

I'm in no way invested in the wording though.

**Comment by Jared-Fogle** [comment]
Map diff bot isn't your fault, not sure what's up with it.

**Review by Mothblocks — APPROVED** [review]

File: code/datums/mood_events/generic_negative_events.dm [hunk 2]
```diff
@@ -310,3 +310,8 @@
 	description = "<span class='warning'>All the fish are dead...</span>\n"
 	mood_change = -3
 	timeout = 90 SECONDS
+
+/datum/mood_event/burnt_wings
+	description = "<span class='boldwarning'>MY PRECIOUS WINGS!!</span>\n"
+	mood_change = -10
+	timeout = 10 MINUTES
```
Comment (ATH1909) [inline code/datums/mood_events/generic_negative_events.dm:317]: this is a bit, uh, harsh, no? this is basically gonna tank your mood (and thus your sanity) into the gutter unless you immediately drop everything to pursue as many positive moodlets as possible.

Comment (Jared-Fogle) [reply code/datums/mood_events/generic_negative_events.dm:317]: Correct! It's mechanically reinforcing roleplay in a way I'd like to see more of.

Comment (CRITAWAKETS) [reply code/datums/mood_events/generic_negative_events.dm:317]: bro your organs melted

**Comment by Mickyan** [comment]
This should be cleared by reconstruction surgery

**Comment by ArcaneDefence** [comment]
> 
> 
> This should be cleared by reconstruction surgery

Dismemberment moodlets in general aren't cleared by surgery, even if reattaching the same limb immediately after severing it.
It even reads "AHH! I WAS USING THAT LIMB!"

File: code/datums/mood_events/generic_negative_events.dm (hunk 2 above)
Comment (ArcaneDefence) [reply code/datums/mood_events/generic_negative_events.dm:317]: Good.

**Comment by Jared-Fogle** [comment]
You have trauma now, knowing how vulnerable your precious wings are at any moment. Thus you are still sad.

**Pull request merged**